app/controller.js

```diff
@@ -87,6 +88,21 @@ class Controller {
     }
   }
 
+  async addAddress(req, res) {
```

Comment:
If possible, rename the method to something more "descriptive". Otherwise, code looks fine.